### DISCUSSION TITLE
How to configure kerberos when submit spark task to kerberized cluster?

### DISCUSSION THREAD

**Discussion by turtoise** [apache/dolphinscheduler#12275] [Q&A]
Here is my question,when i submit a  spark task to kerberized hadoop cluster,i don't know how to configure kerberos to authorize.

this is my configuration:
![image](https://user-images.githubusercontent.com/19393229/194741771-0e589cd1-475e-4609-9369-40e9fd7b3c7a.png)

But it report an error like this:
![image](https://user-images.githubusercontent.com/19393229/194741834-b44c73da-dfec-4266-aad8-769708215fd4.png)

**Comment by wsczm** [comment]
你好 请问这个问题解决了 遇到了同样的问题

**Comment by ziya05** [comment]
I have same problem, do you have fixed this problem please?
